constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java

```diff
@@ -31,12 +31,12 @@ public class MotionWidget implements TypedValues {
 
     public static final int VISIBILITY_MODE_NORMAL = 0;
     public static final int VISIBILITY_MODE_IGNORE = 1;
-    @SuppressWarnings("unused") private static final int INTERNAL_MATCH_PARENT = -1;
-    @SuppressWarnings("unused") private static final int INTERNAL_WRAP_CONTENT = -2;
+    private static final int INTERNAL_MATCH_PARENT = -1;
+    private static final int INTERNAL_WRAP_CONTENT = -2;
     public static final int INVISIBLE = 0;
     public static final int VISIBLE = 4;
-    @SuppressWarnings("unused") private static final int INTERNAL_MATCH_CONSTRAINT = -3;
-    @SuppressWarnings("unused") private static final int INTERNAL_WRAP_CONTENT_CONSTRAINED = -4;
```

Comment:
It doesn't seem like these are being used.

Reply:
This was a rebase accident removing the changes